core/block/simple/dataview/views.go

```diff
@@ -207,3 +209,11 @@ func (l *Dataview) ReorderViewRelations(viewID string, relationKeys []string) er
 	}
 	return nil
 }
+
+func (l *Dataview) setRelationFormat(filter *model.BlockContentDataviewFilter) {
```

Comment:
We are setting relation format explicitly, because format is not obligatory field in protocol and longtext=0 was set by default